frontend/catalyst/qjit_device.py

```diff
@@ -160,19 +167,19 @@ def __init__(
         self.backend_kwargs = backend.kwargs if backend else {}
         device_name = backend.device_name if backend else "default"
 
-        shots_present = shots is not None
-        self._operations = get_qjit_pennylane_operations(target_config, shots_present, device_name)
-        self._observables = get_pennylane_observables(target_config, shots_present, device_name)
+        program_features = ProgramFeatures(shots is not None)
+        target_device_config = get_device_config(target_config, program_features, device_name)
+        self.caps = get_qjit_device_config(target_device_config)
 
     @property
     def operations(self) -> Set[str]:
-        """Get the device operations"""
-        return self._operations
+        """Get the device operations using PennyLane's syntax"""
+        return pennylane_operation_set(self.caps.native_ops)
 
     @property
     def observables(self) -> Set[str]:
         """Get the device observables"""
-        return self._observables
+        return pennylane_operation_set(self.caps.native_obs)
```

Comment:
This is not covered.

Reply:
[Fixed](https://github.com/PennyLaneAI/catalyst/blob/8239b368b9cc1fdcb2b458b2d87b06bcbee339a2/frontend/test/pytest/test_config_functions.py#L396)